src/common/components/PageLoading.jsx

```diff
@@ -1,11 +1,8 @@
 import React from 'react';
 
-import Banner from './Banner';
```

Comment:
Banner is profile-specific, but PageLoading isn't.  I wanted to use PageLoading in the MFE boilerplate, so I split out Banner and put it in place in the profile while it's loading.  

This isn't perfect, as we probably want the banner showing during _all_ preloaders, but that'd be a deeper refactor of our data loading in this app.